### DISCUSSION TITLE
Welcome to gwt-eclipse-plugin Discussions!

### DISCUSSION THREAD

**Discussion by niloc132** [gwt-plugins/gwt-eclipse-plugin#432] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by protoism** [comment]
Here's Luca,
owner of a small company which happily works with GWT, ignoring all sorts of modern frameworks.
And yes, we still use Eclipse, and like it.
We like it so much, that I felt that we had to try and contribute to reviving this project.

**Reply by jetmore** [reply]
Small dev team working on an app that has been using GWT with eclipse since 2009.  Very happy with the product but lots of little paper cuts have led us to consider rewriting the front end entirely, which we really don't want to do.  Happy to see life in this project and hope to see more and contribute when it becomes easier to understand where and how
